src/autodiff.jl

```diff
@@ -38,6 +38,17 @@ function _init_seed!(t1sseeds, coloring, ncolor, nmap)
     end
 end
 
+function transfer!(target::AbstractJacobian, origin::AbstractJacobian)
+    if target.x isa Array
```

Comment:
I think we could use multiple dispatch there. But IMO, it would make more sense to update the code once the PR about AD refactoring is merged